apps/studio/src/server/modules/audit/audit.service.ts

```diff
@@ -0,0 +1,181 @@
+import type {
+  AuditLogEvent,
+  Blob,
+  DB,
+  Resource,
+  ResourcePermission,
+  Transaction,
+  User,
+  VerificationToken,
+} from "../database"
+
+type FullResource = Resource & (Blob | undefined)
```

Comment:
question: seems like we might be omitting changes to site config, navbar and footer? Or should they be captured as a separate AuditLogEvent?

Reply:
oh good catch - for now, we don't have user capability to add it in so it'll have to be through the admin endpoint. i think it'll be a different `AuditLogEvent` because we want to capture the full db item